### PR TITLE
Merge some item groups

### LINKED ISSUE #169: If you don't care about/have time for #168.
Temporary workaround till #168 is (hopefully) resolved.
Tabs which should be merged together. 
Tools with Weapons (they are tools too so why isn't this together at first place.
image
image

Armor with Accessories, under the name Wearables
image
image

There is also: Misc, Foods, Materials, Blocks.
Maybe Foods + Misc?

### PR DESCRIPTION
**Pull request by nailuj29gaming**
Closes #169

### CODE REVIEW

**Comment by nailuj29gaming** [comment]
One thing of note: "Aether Wearables" doesn't have any translations other than English

File: src/main/java/com/aether/items/AetherItemGroups.java [hunk 1]
```diff
@@ -15,21 +15,21 @@
             Aether.locate("aether_tools"),
             () -> new ItemStack(AetherItems.GRAVITITE_PICKAXE));
 
-    public static final ItemGroup Weapons = FabricItemGroupBuilder.build(
-            Aether.locate("aether_weapons"),
-            () -> new ItemStack(AetherItems.GRAVITITE_SWORD));
+//    public static final ItemGroup Weapons = FabricItemGroupBuilder.build(
+//            Aether.locate("aether_weapons"),
+//            () -> new ItemStack(AetherItems.GRAVITITE_SWORD));
```
Comment (Haven-King) [inline src/main/java/com/aether/items/AetherItemGroups.java:20]: Code should be removed, not commented out.

**Comment by nailuj29gaming** [comment]
I'm bad at squashing, so you can probably do it when you merge the changes

**Comment by MBatt1** [comment]
So this merges tools into weapons, and armor into accessories?

**Comment by nailuj29gaming** [comment]
Yes

**Pull request merged**